Added support for parameter annotations.  Two listeners are implemented,

dbennett455 opened the pull request:
a listener that accepts only String values in the spirit of the original
listeners plus a method parameter annotation listener that accpets the
Javassist objects for access to a method parameter annotation's
parameters as well as other low-level Javassist info.

Apologies for not including this in the last commit.  I didn't realize the method parameter annotation functionality was missing until after the last changes were committed.

dbennett455 committed:
Added support for parameter annotations.  Two listeners are implemented,
a listener that accepts only String values in the spirit of the original
listeners plus a method parameter annotation listener that accpets the
Javassist objects for access to a method parameter annotation's
parameters as well as other low-level Javassist info.

ngocdaothanh commented:
Thanks!
I've just released v1.6 and updated Javadoc (http://ngocdaothanh.github.io/annovention/).

dbennett455 commented:
Updated README.rst too.

On 02/15/2014 07:53 PM, Ngoc Dao wrote:

> Thanks!
> I've just released v1.6 and updated Javadoc
> (http://ngocdaothanh.github.io/annovention/).
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ngocdaothanh/annovention/pull/6#issuecomment-35174880.

dbennett455 commented:
Looks good,  Thanks!

I created a repo with the test code that pulls from maven.

  https://github.com/dbennett455/annovention-test

--Dave

On 02/15/2014 07:53 PM, Ngoc Dao wrote:

> Thanks!
> I've just released v1.6 and updated Javadoc
> (http://ngocdaothanh.github.io/annovention/).
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ngocdaothanh/annovention/pull/6#issuecomment-35174880.
